spec/like_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+describe User, type: :model do
```

Comment:
- [ ] Kindly ensure your tests are descriptive. This is the Like test file but you seem to be describing User (`describe User`). Kindly fix that.

- **[OPTIONAL]** I would recommend you move these model test files to a new `models` directory. Instead of leaving them at the spec folder you can create a `spec/models` folder and move the tests there.